EIPS/eip-4200.md

```diff
@@ -148,35 +143,35 @@ def validate_code(code: bytes):
         # Ensure the opcode is valid
         opcode = code[pos]
         pos += 1
-        assert(opcode in valid_opcodes)
+        assert opcode in valid_opcodes
 
         if opcode == 0x5c or opcode == 0x5d:
-            assert(pos + 2 <= len(code))
+            assert pos + 2 <= len(code)
             offset = int.from_bytes(code[pos:pos+2], byteorder = "big", signed = True)
 
             rjumpdest = pos + 2 + offset
-            assert(rjumpdest >= 0 and rjumpdest < len(code))
+            assert rjumpdest >= 0 and rjumpdest < len(code)
 
             rjumpdests.add(rjumpdest)
 
-        # Save immediate value positions
+        # Save positions of each byte of immediate value
```

Comment:
Clarification requested in https://github.com/ipsilon/eof/pull/3#discussion_r801517169